pkg/backend/benchmark_test.go

```diff
@@ -102,14 +102,30 @@ func BenchmarkRenders(b *testing.B) {
 }
 
 func BenchmarkRendersStorm(b *testing.B) {
```

Comment:
The current benchmark only has 3 metrics and the production issue that we are seeing usually have a higher number of metrics.

Maybe we can introduce a benchmark function that has high metric cardinality and relatively high number of metrics.

6000 metrics * 3 replicas * 10800 data points, not sure if it's too much.

Reply:
Yeah that makes more sense.

Reply:
made the change.